infra-sofa-boot-starter/src/main/java/com/alipay/sofa/infra/config/spring/namespace/handler/SofaBootNamespaceHandler.java

```diff
@@ -38,18 +39,16 @@
     @Override
     public void init() {
         ServiceLoader<SofaBootTagNameSupport> serviceLoaderSofaBoot = ServiceLoader
-            .load(SofaBootTagNameSupport.class);
-        //SOFABoot
-        for (SofaBootTagNameSupport tagNameSupport : serviceLoaderSofaBoot) {
-            this.registerTagParser(tagNameSupport);
-        }
+                .load(SofaBootTagNameSupport.class);
+        StreamSupport.stream(serviceLoaderSofaBoot.spliterator(), false)
+                .forEach(this::registerTagParser);
```

Comment:
Why not use `serviceLoaderSofaBoot.forEach(it -> registerTagParser(it))` here? I think it's more clear and simple.

Reply:
@straybirdzls  fixed.